SUI/SUI/BasketView.swift

```diff
@@ -0,0 +1,120 @@
+    }
+    
+    // MARK: - Public Properties
+    
+    var body: some View {
+        VStack {
+            titleTextView
+            Spacer()
+            descriptionTextView
+            typeTextView
+            Spacer()
+            VStack {
+                priceTitleTextView
+                priceTextView
+            }
+            Spacer()
+            buyButtonView
+            cancelButtonView
+            Spacer(minLength: Constants.minSpacerNumber)
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    @EnvironmentObject private var userBuy: UserBuyRateViewModel
+    
+    @StateObject private var viewModel = BasketViewModel()
+
+    private var titleTextView: some View {
+        Text(Constants.titleString)
+            .font(.largeTitle)
+            .bold()
+            .baselineOffset(Constants.titleBaselineOffsetNumber)
+    }
+
+    private var descriptionTextView: some View {
+        Text(Constants.typeString)
+            .baselineOffset(Constants.typeBaselineOffsetNumber)
+            .font(.system(size: Constants.fontSizeNumber, weight: .light, design: .default))
+    }
+    
+    private var typeTextView: some View {
+        Text(userBuy.type)
+            .baselineOffset(Constants.typeBaselineOffsetNumber)
+            .font(.title)
+    }
+    
+    private var priceTitleTextView: some View {
+        Text(Constants.priceString)
+            .baselineOffset(Constants.typeBaselineOffsetNumber)
+            .font(.system(size: Constants.fontSizeNumber, weight: .light, design: .default))
+    }
+    
+    private var priceTextView: some View {
+        Text(userBuy.price)
+            .baselineOffset(Constants.typeBaselineOffsetNumber)
+            .font(.title)
+    }
+    
+    private var cancelButtonView: some View {
+        Button {
+            userBuy.badge = Constants.badgeNumber
+            userBuy.type = Constants.emptyString
+            userBuy.price = Constants.emptyString
+        } label: {
+            Text(Constants.clearButtonTitle)
+                .foregroundColor(.white)
+                .font(.headline)
+                .padding()
+                .frame(width: Constants.widthButtonNumber, height: Constants.heightButtonNumber, alignment: .center)
+                .background(.blue)
+                .cornerRadius(Constants.cornerRadiusButtonNumber)
+        }
+    }
+    
+    private var buyButtonView: some View {
+        Button {
+            guard !userBuy.type.isEmpty else { return }
+            viewModel.isBuyButtonPressed = true
+        } label: {
+            Text(Constants.buyButtonTitle)
+                .foregroundColor(.white)
+                .font(.headline)
+                .padding()
+                .frame(width: Constants.widthButtonNumber, height: Constants.heightButtonNumber, alignment: .center)
+                .background(.blue)
+                .cornerRadius(Constants.cornerRadiusButtonNumber)
+        }.alert(isPresented: $viewModel.isBuyButtonPressed) {
```

Comment:
```suggestion
        }
        .alert(isPresented: $viewModel.isBuyButtonPressed) {
```
Все модифаеры с новой строки пишем